setup.py

```diff
@@ -1,8 +1,21 @@
+import platform
+import sys
+
 from setuptools import setup, Extension
 import setuptools_scm  # noqa  Ensure it’s installed
 
+# Check for 64-bit
+if sys.maxsize > 2**32 and (
```

Comment:
If `platform.machine()` already says "x86_64", it should not be necessary to check whether the platform is 64 bit.

Reply:
Agreed. Done!